aws/aws.go

```diff
@@ -65,7 +65,7 @@ func ReadFile(bucketName string, filename string, outFile string, noColors bool)
 		log.Fatal(style.Bold(style.Red("ERROR: Failed to download file  " + filename + " from S3: " + err.Error())))
 	}
 
-	log.Println("INFO: Successfully downloaded " + filename + " from S3 as " + outFile)
+	log.Println("Successfully downloaded " + filename + " from S3 as " + outFile)
```

Comment:
Should this be `logs.Info` instead of `log.Println` ?

Reply:
since this is inside of aws package, I can't import logs from root in a clean way, so decided to let them be for now as they are